tests/utils_tests/test_dateformat.py

```diff
@@ -165,3 +165,7 @@ def test_r_format_with_non_en_locale(self):
                 dateformat.format(dt, 'r'),
                 'Sun, 08 Jul 1979 22:00:00 +0100',
             )
+
+    def test_format_before_1000(self):
+        birthday_of_aryabhata = datetime(476, 9, 8, 5, 00)
+        self.assertEqual(dateformat.format(birthday_of_aryabhata, "d/m/y"), "08/09/76")
```

Comment:
We should also a test for leading zeros, e.g. `datetime(4, 9, 8, 5, 00)`.

Reply:
Okay, will add it.